NVIDIA.Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.12.5-bookworm
+
+RUN pip install jinja2 numpy
+RUN pip install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/rocm6.0
+#RUN pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu121
```

Comment:
The nvidia dockerfile would need the CUDA URL

Reply:
You are right, my bad. I only changed the commented unstable url

Reply:
Fixed